eth/statedb/config.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 Evmos Foundation
```

Comment:
Let's create an interface for the TxConfig in the StateDB struct, then remove this implementation and keep it in etheremint